SDWebImage/SDWebImageManager.m

```diff
@@ -340,20 +347,34 @@ - (void)callCompletionBlockForOperation:(nullable SDWebImageCombinedOperation*)o
 @end
 
 
+@interface SDWebImageCombinedOperation ()
```

Comment:
Replace @synchronized lock with dispatch_semaphore lock for SDWebImageCombinedOperation.